package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo-drivex",
-  "version": "0.3.0",
+  "version": "0.4.4",
```

Comment:
The patch number doesn't have to match. When there's no major version # it's just the minor version that has to match in terms of peerDependencies.